### PR TITLE
tests: print and store time spent for tests and test sets

### PR DESCRIPTION
**Pull request by yhr**
It's useful to know how long a test runs, both for scheduling
testing and to compare different runs to look for regressions
and improvements in execution time.

So, print the time it takes to run each test and the total
time to run a whole test set in run.sh. Also store the time in the results
directory for comparing different runs.

Signed-off-by: Hans Holmberg <hans.holmberg@wdc.com>

### CODE REVIEW

**Comment by MaisenbacherD** [comment]
Looks good to me.

File: tests/run.sh [hunk 1]
```diff
@@ -18,20 +27,24 @@ RESULT_DIR="results/$NAME/$TEST_DIR"
 mkdir -p $RESULT_DIR
 rm -rf $RESULT_DIR/*
 
+SECONDS=0
```
Comment (metaspace) [inline tests/run.sh:30]: Could you make a comment (maybe with a reference to the manual) about how this variable works?

**Review by metaspace — CHANGES_REQUESTED** [review]

File: tests/run.sh (hunk 1 above)
Comment (yhr) [reply tests/run.sh:30]: yeah, it's a bit obscure

Comment (yhr) [reply tests/run.sh:32]: fixed

**Review by metaspace — APPROVED** [review]

**Comment by yhr** [comment]
Rebased

**Pull request merged**